Firestore/Example/Tests/Local/StringViewTests.mm

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <leveldb/slice.h>
+#import "Firestore/Source/Local/StringView.h"
```

Comment:
StringView header should be first (to ensure it's self contained.

Reply:
It was... Oh I see, style.sh "fixed" it. I've added a blank line to ensure it doesn't do so again.